tools/c7n_azure/c7n_azure/resources/disk.py

```diff
@@ -50,3 +50,14 @@ class resource_type(ArmResourceManager.resource_type):
             'sku.name'
         )
         resource_type = 'Microsoft.Compute/disks'
+
+@Disk.action_registry.register('snapshot')
```

Comment:
Hey @nebed, could you add a test case for this? Because it is a new action, there is currently no code coverage on it.